actors/miner/tests/prove_commit2_failures_test.rs

```diff
@@ -25,22 +25,22 @@ const FIRST_SECTOR_NUMBER: SectorNumber = 100;
 #[test]
 fn reject_unauthorized_caller() {
     let (h, rt, activations) = setup_precommits(&[(0, 0, 0)]);
-    let cfg = ProveCommitSectors2Config {
```

Comment:
Rename the file too? Perhaps a separate PR though (ahead of this one?)

Reply:
Tracking follow on in issue comment [here](https://github.com/filecoin-project/builtin-actors/issues/1545#issuecomment-2131566309)